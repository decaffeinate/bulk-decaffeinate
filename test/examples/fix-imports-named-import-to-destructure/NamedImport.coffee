{twelve, seventeen: tenAndSeven} = require './TwoValueDefaultExport'
{twenty} = require './NameClash'
{twentySix} = require './NamedExport';
{thirtyOne} = require './dashed-name'
console.log twelve;
console.log tenAndSeven;
NameClash = 25
console.log twenty;
console.log twentySix;
console.log NameClash;
console.log thirtyOne;
