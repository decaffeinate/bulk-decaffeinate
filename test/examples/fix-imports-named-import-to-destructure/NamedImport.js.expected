// TODO: This file was created by bulk-decaffeinate.
// Sanity-check the conversion and remove this comment.
import TwoValueDefaultExport from './TwoValueDefaultExport';

const {
  twelve,
  seventeen: tenAndSeven,
} = TwoValueDefaultExport;

import NameClash1 from './NameClash';

const {
  twenty,
} = NameClash1;

import { twentySix } from './NamedExport';
import DashedName from './dashed-name';

const {
  thirtyOne,
} = DashedName;

console.log(twelve);
console.log(tenAndSeven);
let NameClash = 25;
console.log(twenty);
console.log(twentySix);
console.log(NameClash);
console.log(thirtyOne);
